packages/zipkin/src/batch-recorder.js

```diff
@@ -135,6 +135,15 @@ class BatchRecorder {
     }
   }
 
+  /**
+   * Calling this will flush any pending spans to the transport.
```

Comment:
```suggestion
   * Calling this will flush any pending spans to the transport.
   *
   * Note: the transport itself may be batching, in such case you may need to flush that also.
```